cli/command/image/build.go

```diff
@@ -156,13 +156,13 @@ func runBuild(dockerCli *command.DockerCli, options buildOptions) error {
 
 	switch {
 	case specifiedContext == "-":
-		buildCtx, relDockerfile, err = builder.GetContextFromReader(dockerCli.In(), options.dockerfileName)
+		buildCtx, relDockerfile, err = buildutils.GetContextFromReader(dockerCli.In(), options.dockerfileName)
```

Comment:
@tonistiigi if it's only used in `cli` code let's put the function there then (`cli/commands/images/build` or `cli/build` for example).